api/billingd/service/service.go

```diff
@@ -827,20 +828,20 @@ func (s *Service) GetCustomerUsage(
 	usage := make(map[string]*pb.Usage)
 	for k, u := range doc.DailyUsage {
 		if product, ok := s.products[k]; ok {
-			// Get reported usage over the current invoice period
-			free, err := s.getPeriodUsageItem(u.FreeItemID)
-			if err != nil {
-				return nil, err
-			}
-			paid, err := s.getPeriodUsageItem(u.PaidItemID)
-			if err != nil {
-				return nil, err
-			}
-			total := (free.TotalUsage + paid.TotalUsage) * product.UnitSize
 			// Add current day unreported usage
+			total := u.Total
+			// Get reported usage over the current invoice period
 			if product.FreeQuotaInterval == FreeQuotaDaily &&
 				product.PriceType == PriceTypeIncremental {
-				total += u.Total
+				free, err := s.getPeriodUsageItem(u.FreeItemID)
+				if err != nil {
+					return nil, err
+				}
+				paid, err := s.getPeriodUsageItem(u.PaidItemID)
+				if err != nil {
+					return nil, err
+				}
+				total += (free.TotalUsage + paid.TotalUsage) * product.UnitSize
```

Comment:
This logical will have to change when I add the usage by date range functionality, but this works for the current billing period (month).